assignments1/calculateFrequency.js

```diff
@@ -0,0 +1,16 @@
+function calculateFrequency(string) {
+  
+  var freq={};
+  for(var iter=0;iter<string.length;iter++)
+    {
+         var ch=string.charAt(iter);
```

Comment:
- RIP indentation! Always use 2 spaces indentation for blocks.
- `ch` ?? Name the variable according to what it stores!